package.json

```diff
@@ -18,6 +18,7 @@
   },
   "precommit": [
     "lint",
+    "type-check",
     "test"
```

Comment:
Nice. Didn’t know that the npm tooling provides a mechanism to automatically enable git hooks 👌